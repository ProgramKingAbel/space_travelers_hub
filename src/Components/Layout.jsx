import React from 'react';
import { Outlet } from 'react-router-dom';


const Layout = () => {
  return (
    <div>Layout</div>
  )
}

export default Layout