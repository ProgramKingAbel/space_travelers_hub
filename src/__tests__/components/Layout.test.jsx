import React from 'react';
import { MemoryRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Layout from '../../Components/Layout';

describe('Layout', () => {
  test('renders correctly', () => {
    const tree = renderer
      .create(
        <MemoryRouter>
          <Layout />
        </MemoryRouter>,
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
