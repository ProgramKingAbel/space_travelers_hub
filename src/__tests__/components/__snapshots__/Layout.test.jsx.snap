// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Layout renders correctly 1`] = `
<div>
  <nav
    className="nav"
  >
    <div
      className="logoDiv"
    >
      <img
        alt="space-travelers-hub"
        className="logo"
        src="planet.png"
      />
      <h1>
        Space Tavelers' Hub
      </h1>
    </div>
    <ul
      className="links"
    >
      <li>
        <a
          aria-current="page"
          className="link active"
          href="/"
          onClick={[Function]}
        >
          Rockets
        </a>
      </li>
      <li>
        <a
          className="link"
          href="/Dragons"
          onClick={[Function]}
        >
          Dragons
        </a>
      </li>
      <li>
        <a
          className="link"
          href="/Missions"
          onClick={[Function]}
        >
          Missions
        </a>
      </li>
      <li>
        <a
          className="link"
          href="/Profile"
          onClick={[Function]}
        >
          My Profile
        </a>
      </li>
    </ul>
  </nav>
</div>
`;
